lib/tasks/db.js

```diff
@@ -113,7 +113,7 @@ module.exports = {
 
     task: function () {
       getMigrator('seeder', function (migrator) {
-        return migrator.pending()
+        return migrator.executed()
```

Comment:
We need to handle both cases, for the case when there is no storage, pending is necessary, else executed is correct.

```javascript
        return (helpers.umzug.getStorage('seeder') === 'none'
             ? migrator.pending()
             : migrator.executed())
```